src/p_maputl.c

```diff
@@ -419,13 +419,123 @@ boolean P_BlockLinesIterator(int x, int y, boolean func(line_t*))
 //
 // killough 5/3/98: reformatted, cleaned up
 
+boolean blockmapfix;
+extern boolean PIT_RadiusAttack(mobj_t* thing);
```

Comment:
We should probably also add:

`extern boolean PIT_ChangeSector(mobj_t* thing);`